src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -817,6 +817,12 @@ function getTransformHandles(transformBounds, margin) {
 }
 
 function getTransformSelectionBounds(glyph, selection, getBackgroundImageBoundsFunc) {
+  if (selection.size == 1 && parseSelection(selection).point?.length == 1) {
+    // Return if only one point is selected.
+    // NOTE: We need to check selection.size == 1 because otherwise we don't know
```

Comment:
This comment no longer fully makes sense with the current code.

Reply:
Mmhhh, for me it still makes sense. But I am happy to change it if you have any other suggestions.

Reply:
The comment suggests the `selection.size` check is "extra", but in the code it is the primary check. It is explaining things that should be obvious if you know how the selection set works, and how parseSelection() works. This is not the place to explain those concepts.

I think the whole comment should read:
```js
    // Return if only a single point is selected, as in that case the "selection bounds"
    // is not really useful for the user, and is distracting instead.
```